changelogs/fragments/syslog_exception.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
Sorry shouldn't this be a bugfixe not change, I assume we will be backporting to 2.16 for Python 3.12 support there?